components/ProductListItem.vue

```diff
@@ -82,55 +81,45 @@ export default {
       height: 200px;
     }
   }
-
   .card-content {
     width: 100%;
   }
-
-  .title, .subtitle {
-    color: inherit;
-  }
-
   .title {
     margin-bottom: 0.5rem;
   }
-
   .button {
     border: 0;
     padding: 0;
-
     .fa-circle {
       transition: color 0.5s;
     }
-
     .fa-cart-plus {
       font-size: 1.4rem;
     }
-
     &:hover {
       .fa-circle {
         color: #209cee;
       }
     }
-
     &.icon {
       cursor: pointer;
     }
   }
-
   a {
     color: inherit;
-
     &:hover {
       color: #3273dc;
     }
   }
 }
-
-.lazyload, .lazyloading {
+.card .title,
+.card .subtitle {
```

Comment:
Why was it moved from .card definition? Was something breaking?

Reply:
Oops, I think I messed up here. Good catch!